fastmot/utils/visualization.py

```diff
@@ -1,10 +1,17 @@
 import colorsys
 import numpy as np
 import cv2
-
+from .rect import get_center
 
 GOLDEN_RATIO = 0.618033988749895
 
+def draw_trajectories(frame, tracks):
+    for track in tracks:
+        tlbrs = np.reshape(list(track.bboxes), (len(track.bboxes), 4))
+        centers = tuple(map(lambda box: get_center(box), tlbrs[::10]))
```

Comment:
Why skip by 10 here? Is it faster, or does it look cleaner?

Reply:
It is a bit faster when you increase the skip number but in general the drawing doesn't change a lot unless the skip is way much bigger.